lib/logstash/outputs/elasticsearch/common.rb

```diff
@@ -281,6 +290,7 @@ def safe_bulk(actions)
       sleep_interval = @retry_initial_interval
       begin
         es_actions = actions.map {|action_type, params, event| [action_type, params, event.to_hash]}
+        @logger.error("There are #{actions.count} actions to do")
```

Comment:
Is this an error situation?

Reply:
@guyboertje I think you are looking at an outdated commit